### PR TITLE
(GH-198) Blog post about the three new email addins

### PR DESCRIPTION
**Pull request by Jericho**

### CODE REVIEW

File: src/Cake.Web/App_Data/blog/2016-10-31-email-addins-released.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
+---
+title: Email addins released
+category: Release Notes
```
Comment (gep13) [inline src/Cake.Web/App_Data/blog/2016-10-31-email-addins-released.md]: I think Announcement would be better suited.  Release Notes is really reserved for Cake Releases.

File: src/Cake.Web/App_Data/blog/2016-10-31-email-addins-released.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+author: jericho
+---
+
+There are many scenarios where you need to send notifications during your build process, such as notifying colleagues that a build has passed/failed, notifying stakeholders that a package has been published to nuget, etc. One of the most popular ways to send such notification is, of course, email. That's why we recently released three new addins to send emails.
+
+__Cake.Email__
```
Comment (gep13) [inline src/Cake.Web/App_Data/blog/2016-10-31-email-addins-released.md]: Can you make this a link to the NuGet Package?

Also, can you add a link within the paragraph to the source repository?

File: src/Cake.Web/App_Data/blog/2016-10-31-email-addins-released.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    }
+    catch(Exception ex)
+    {
+        Error("{0}", ex);
+    }
+});
+```
+
+
+__Cake.SendGrid__
```
Comment (gep13) [inline src/Cake.Web/App_Data/blog/2016-10-31-email-addins-released.md]: Can you make this a link to the NuGet Package?

Also, can you add a link within the paragraph to the source repository?

File: src/Cake.Web/App_Data/blog/2016-10-31-email-addins-released.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+        }
+    }
+    catch(Exception ex)
+    {
+        Error("{0}", ex);
+    }
+});
+```
+
+__Cake.CakeMail__
```
Comment (gep13) [inline src/Cake.Web/App_Data/blog/2016-10-31-email-addins-released.md]: Can you make this a link to the NuGet Package?

Also, can you add a link within the paragraph to the source repository?

File: src/Cake.Web/App_Data/blog/2016-10-31-email-addins-released.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+author: jericho
+---
+
+There are many scenarios where you need to send notifications during your build process, such as notifying colleagues that a build has passed/failed, notifying stakeholders that a package has been published to nuget, etc. One of the most popular ways to send such notification is, of course, email. That's why we recently released three new addins to send emails.
+
+__Cake.Email__
+
+The first addin is caled Cake.Email and allows you to send emails via your own SMTP server. 
+
+When invoking the `Email.SendEmail` alias, you must specify the name and email address of the sender (this is what the recipient will see in the `from` field when they receive your eamil). Typically, you would use your name and email address but you could use a generic address such as `notifications@yourcompany.com`), the name and email address of the recipient, the subject and content of the email, a boolean value indicating if the content should be sent as HTML or plain text and finally you must also specify the settings that are necessary to establish a connection to your SMTP server.
```
Comment (devlead) [inline src/Cake.Web/App_Data/blog/2016-10-31-email-addins-released.md]: typo `eamil`

File: src/Cake.Web/App_Data/blog/2016-10-31-email-addins-released.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+    }
+});
+```
+
+
+__Cake.SendGrid__
+
+The second addin allows you to send emails via a third-party Email Service Provider (ESP) called [SendGrid](https://sendgrid.com). The parameters you pass to the `SendGrid.SendEmail` alias are very similar to what we described earlier except that it accepts both the HTML and the plain text versions which will be sent in the same "multi-part" email, and also the only setting expected by SendGrid is your API Key.
+
+Here's an exmaple:
```
Comment (devlead) [inline src/Cake.Web/App_Data/blog/2016-10-31-email-addins-released.md]: Typo `exmaple`

**Pull request merged**